tests/serialization_tests.rs

```diff
@@ -1662,6 +1667,12 @@ symmetric_tests! {
         be = [0, 0, 0, 3, 1, 2, 3],
         minimum_bytes = 4
     }
+    derived_struct_with_generic_default for DerivedStructWithGenericDefault {
+        in = DerivedStructWithGenericDefault { inner: 1 },
+        le = [1],
+        be = [1],
+        minimum_bytes = 1
+    }
```

Comment:
Can you also add one more test where the default param is overriden?

Reply:
ofcourse, done!